### DISCUSSION TITLE
Are vulnerabilities fixed in the github repo?

### DISCUSSION THREAD

**Discussion by josefholmner** [eProsima/Fast-DDS#3816] [General]
Sequrity vulnerabilities found in e.g. Ubuntu DDS packages such as: https://ubuntu.com/security/notices/USN-6306-1 , are these fixed in the public github repo? How serious does the develipers of this repository take these reports? Are there any plans for the above currently?

**Comment by JLBuenoLopez** [comment]
@josefholmner 

You can look into the [security tab of the repository](https://github.com/eProsima/Fast-DDS/security/advisories?state=published). The six security vulnerabilities reported in the linked issue have already been fixed in master and backported to the [alive and maintained branches of Fast DDS](https://github.com/eProsima/Fast-DDS/blob/master/RELEASE_SUPPORT.md) (v2.10.x, v2.6.x).

**Reply by josefholmner** [reply]
Thank you for that clarification
